glusterfs.spec.in

```diff
@@ -53,6 +53,16 @@
 # rpmbuild -ta @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --with ipv6default
 %{?_with_ipv6default:%global _with_ipv6default --with-ipv6-default}
 
+# linux-io_uring
+# If you wish to compile an rpm without linux-io_uring support...
+# rpmbuild -ta  @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --disable-linux-io_uring
+%{?_without_linux_io_uring:%global _without_linux_io_uring %--disable-linux-io_uring}
+
+# Disable linux-io_uring on unsupported distros.
+%if ( 0%{?rhel}) || ( 0%{?fedora} && 0%{?fedora} < 32 )
+%global _without_linux_io_uring ---disable-linux-io_uring
+%endif
+
```

Comment:
@nixpanic  So the smoke tests still fail with the spec file changes. Please see if the changes I made make sense.